Parser: Convert power data to number format

As a result of #40, we need to also correct the data in the parser to also be a number for consistency's sake.

```

    src/utils/factory-management/mocks/mockRecipes.ts:226:7 - error TS2322: Type 'string' is not assignable to type 'number'.

    226       power: '1.5',
              ~~~~~

      src/interfaces/Recipe.ts:15:5
        15     power: number;
               ~~~~~
        The expected type comes from property 'power' which is declared here on type '{ name: string; power: number; }'
```

--- a/parsing/src/interfaces/Recipe.ts
+++ b/parsing/src/interfaces/Recipe.ts
@@ -12,7 +12,7 @@ export interface Recipe {
   products: RecipeItem[];
   building: {
     name: string;
-    power: string;
+    power: number;
   }
   isAlternate: boolean;
   isFicsmas: boolean;

--- a/parsing/src/processor.ts
+++ b/parsing/src/processor.ts
@@ -332,7 +332,7 @@ function getRecipes(
 // Create building object with the selected building and calculated power
             const building = {
                 name: selectedBuilding || '', // Use the first valid building, or empty string if none
-                power: String(powerPerBuilding || 0), // Use calculated power or 0
+                power: powerPerBuilding || 0, // Use calculated power or 0
             };
 
 

